src/installer/installer.go

```diff
@@ -227,6 +260,25 @@ func (i *installer) extractIgnitionToFS(ignitionPath string) (err error) {
 	return err
 }
 
+func (i *installer) generateSshKeyPair() error {
+	i.log.Info("Generating new SSH key pair")
+	if _, err := i.ops.ExecCommand(utils.NewLogWriter(i.log), "ssh-keygen", "-q", "-f", "/root/.ssh/id_rsa", "-N", "''"); err != nil {
```

Comment:
Don't want to block the PR on this, but it feels like there should be a "native" way to generate a ssh keypair in go without using exec.

Then we wouldn't have to write and re-read the file as well.

Reply:
There is but then something else will be needed:
I will need to write the output to `/root/.ssh` so I don't think we will gain anything from it.